### PR TITLE
Add a changelog

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## Unreleased
+
+<!-- New PRs should document their changes here. -->
```
Comment (FredKSchott) [inline CHANGELOG.md]: add a "do not delete this line!" or equivalent so that people don't remove after adding the first unreleased change

Comment (rictic) [inline CHANGELOG.md]: Eh, IMO we can say that in code review if needed.

Comment (FredKSchott) [inline CHANGELOG.md]: happy to start here and revisit later if we're having problems

**Comment by FredKSchott** [comment]
LGTM!

**Comment by rictic** [comment]
Thanks for the review!

**Pull request merged**